pi-geofence/src/androidTest/java/com/ibm/pisdk/geofencing/test/PIGeofencingServiceTest.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2015-2016 IBM Corporation. All rights reserved.
```

Comment:
Bruce said that using a comma is the preferred method for specifying ranges in Copyright years.